src/main/java/software/aws/neptune/jdbc/utilities/ConnectionProperties.java

```diff
@@ -391,6 +396,25 @@ public void setConnectionRetryCount(final int retryCount) throws SQLException {
         put(CONNECTION_RETRY_COUNT_KEY, retryCount);
     }
 
+    /**
+     * Gets the region.
+     *
+     * @return The region.
+     */
+    public String getRegion() {
+        return getProperty(SERVICE_REGION_KEY);
```

Comment:
Would suggest returning:
`return (getProperty(SERVICE_REGION_KEY) == null ? "" : getProperty(SERVICE_REGION_KEY));`
That was if someone passes in null for in their property map for the `SERVICE_REGION_KEY`, we don't get a null pointer exception later if we assume it is non-null.

Reply:
Resolving because I just saw NonNull on the `setRegion` below. Forget this.